### PR TITLE
Initial setup pulseaudio remote connection, control volume from guivm

### PR DESCRIPTION
**Pull request by josa41**
Initial setup for controlling pulseaudio from guivm
- Change volume toggle mute etc working
- Set default max volume level to a bit more reasonable level
- Using Pulseaudio TCP socket over ssh for now

### CODE REVIEW

File: modules/development/ssh.nix [hunk 1]
```diff
@@ -30,6 +31,10 @@ in
     config = mkIf cfg.enable {
       services.openssh.enable = true;
       users.users.root.openssh.authorizedKeys.keys = authorizedKeys;
-      users.users.${config.ghaf.users.accounts.user}.openssh.authorizedKeys.keys = authorizedKeys;
+      users.users.${config.ghaf.users.accounts.user}.openssh.authorizedKeys = {
+        keys = authorizedKeys;
+        # Waypipe-ssh key is used here to create keys for ssh tcp tunneling for pulseaudio.
+        keyFiles = ["${pkgs.waypipe-ssh}/keys/waypipe-ssh.pub"];
```
Comment (mikatammi) [inline modules/development/ssh.nix:37]: The setting from this line should be in targets/lenovo-x1-carbon.nix file

**Review by mikatammi — CHANGES_REQUESTED** [review]
At least one change is needed to prevent breaking NVIDIA builds.

Didn't test anything yet.

File: modules/development/ssh.nix (hunk 1 above)
Comment (josa41) [reply modules/development/ssh.nix]: Thanks, fixed.
Earlier i did not find a correct place for host in lenovo-x1-carbon.nix but now i did.

**Comment by mikatammi** [comment]
@josa41 Please rebase

**Pull request closed without merging**